zerver/webhooks/buildbot/doc.md

```diff
@@ -0,0 +1,11 @@
+Zulip supports integration with CircleCI and can notify you of
+your build statuses.
```

Comment:
Why does it say "CircleCI" here? Shouldn't it be "Buildbot"? Also, I'd recommend rephrasing this as `Get Buildbot notifications in Zulip!`.

Reply:
Sorry about that. I just copied the doc from CircleCI and made some changes there to follow the convention

Reply:
I would recommend changing the introductory line to `Get Zulip notifications for your Buildbot builds!`